<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SheetMusicLibraryDialog</class>
 <widget class="QDialog" name="SheetMusicLibraryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qupil - Noten-Bibliothek</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Alle</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTreeView" name="treeView_all">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Komponist / Autor:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_author"/>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Titel:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_title"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Verlag:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_publisher"/>
           </item>
           <item row="1" column="2" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>258</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <widget class="QPushButton" name="pushButton_addNewSheetMusic">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/qupil.qrc">
               <normaloff>:/gfx/list_add.png</normaloff>:/gfx/list_add.png</iconset>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_delSheetMusic">
             <property name="minimumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>26</width>
               <height>26</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/qupil.qrc">
               <normaloff>:/gfx/editdelete.png</normaloff>:/gfx/editdelete.png</iconset>
             </property>
             <property name="autoExclusive">
              <bool>false</bool>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Verf端gbar</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0" colspan="4">
        <widget class="QTreeWidget" name="treeWidget_available">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Index</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Komponist / Autor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Verlag</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_rentSheetToPupil">
         <property name="text">
          <string>Auswahl verleihen an</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/qupil.qrc">
           <normaloff>:/gfx/document_export.png</normaloff>:/gfx/document_export.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>517</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Sch端ler:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="comboBox_pupils"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Verliehen</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QTreeWidget" name="treeWidget_rentToPupil">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Index</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Komponist / Autor</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Titel</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Verlag</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Verliehen an</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Verliehen am</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_sheetBackFromPupil">
         <property name="text">
          <string>Auswahl von Sch端ler zur端ckgegeben</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/qupil.qrc">
           <normaloff>:/gfx/document_import.png</normaloff>:/gfx/document_import.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>490</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>659</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/qupil.qrc">
       <normaloff>:/gfx/dialog_ok_apply.png</normaloff>:/gfx/dialog_ok_apply.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>treeView_all</tabstop>
  <tabstop>lineEdit_author</tabstop>
  <tabstop>lineEdit_title</tabstop>
  <tabstop>pushButton_addNewSheetMusic</tabstop>
  <tabstop>lineEdit_publisher</tabstop>
  <tabstop>pushButton_delSheetMusic</tabstop>
  <tabstop>treeWidget_available</tabstop>
  <tabstop>pushButton_rentSheetToPupil</tabstop>
  <tabstop>comboBox_pupils</tabstop>
  <tabstop>treeWidget_rentToPupil</tabstop>
  <tabstop>pushButton_sheetBackFromPupil</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/qupil.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SheetMusicLibraryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
